### ISSUE TITLE
Create new static log VM that data can be pushed to

### ISSUE THREAD

**Issue by leifmadsen** [redhat-nfvpe/ansible-cira#30]
Instead of forcing logs to be sent to a directory on the Jenkins master (and subsequently monitored by FileBeat for pushing data to LogStash), look at building a separate VM for static log gathering. Front with nginx and be sure all certificate creation is done, and that things are only available via HTTPS. All inter-VM communication must also be secure.

**Comment by redhat-nfvpe** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;Eu9Bgif6&#x2F;111-create-new-static-log-vm-that-data-can-be-pushed-to"><img src="https:&#x2F;&#x2F;github.trello.services&#x2F;images&#x2F;trello-icon.png" width="12" height="12"> Create new static log VM that data can be pushed to</a>

**Comment by leifmadsen** [comment]
This might get somewhat addressed by my work to build out a separate logging container. I'll file an issue shortly.

**Comment by leifmadsen** [comment]
OK I started working on breaking some stuff out today. The method I was going to use with ARA in a container isn't going to work because the Jenkins slave is not a container on the system. My new approach is now to use a real Nginx container, and to share the contents of the `/srv/static/` directory that is generated on the Jenkins master.

I'll share this as a volume from the Jenkins master, load that into the Nginx (logs) container, and then expose that via Traefik.

**Comment by leifmadsen** [comment]
I'm closing out this issue because I consider ability to read the raw logs to be complete. We're fronting things with NGINX on the Jenkins Master, and in the Docker environment this is now exposed via traefik. I've also fixed some job definition issues that were resulting in problems with ARA. Those are now fixed.

In the future we may revisit this, but for now I'm happy enough with the implementation.
